daemon/container.c

```diff
@@ -537,7 +539,8 @@ container_new(const char *store_path, const uuid_t *existing_uuid, const uint8_t
 					 container_config_get_dns_server(conf) :
 					 cmld_get_device_host_dns();
 
-	list_t *vnet_cfg_list = (ns_net) ? container_config_get_vnet_cfg_list_new(conf) : NULL;
+	list_t *vnet_cfg_list =
+		(ns_net && !priv) ? container_config_get_vnet_cfg_list_new(conf) : NULL;
```

Comment:
maybe veths for c0 necessary?

Reply:
makes no sens with this patchset. Because if a veth is configured for c0 the container endbpoint would be created in c0 and the rootns endpoint would be created in c0 netns.
Thus both end up in c0. In that case c0 can create a veth pair directly by itself.